source/Calamari.Tests/CommandsFixture.cs

```diff
@@ -843,5 +834,73 @@ void IgnoreIfVersionIsNotInRange(string minimum, string? maximum = null)
                 Assert.Ignore($"Test ignored as terraform version is not between {minimumVersion} and {maximumVersion}");
             }
         }
+        
```

Comment:
Note I should be able to bump the test project to net462 as we've done elsewhere and remove this code. We would need to include the Sashimi.Tests.Shared package reference though which isn't ideal...